lib/ravelin.rb

```diff
@@ -1,5 +1,39 @@
-require "ravelin/version"
+require 'date'
```

Comment:
Thanks for the detailed response! I've run out time for this PR but definitely something I'll consider going forward.